packages/website/styles/global.css

```diff
@@ -0,0 +1,12 @@
+.loading {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+  .table-responsive {
+    display: block;
+    width: 100%;
+    overflow-x: auto;
+  }
```

Comment:
_it's the smell_

Reply:
😉 https://github.com/ipfs-shipyard/nft.storage/pull/61#discussion_r618455074